Source/Scene/BoxEmitter.js

```diff
@@ -0,0 +1,45 @@
+/*global define*/
+define([
```

Comment:
@bagnell I would still do this as a primitive; there's still too much plumbing and two APIs here for basically the ability to avoid setting a model matrix and consistency in one sense (when used with entities) and inconsistency in other sense (when used with 3D Tiles).  We could add CZML later if desired.